core/src/apps/cosmos/sign_tx.py

```
@@ -37,17 +37,18 @@ async def sign_tx(
     try:
         tx = Transaction.deserialize(msg.raw_tx)
     except BaseException as e:
-        raise wire.DataError(f"Invalid message {e}")
+        raise wire.DataError(f"Invalid message: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)